### ISSUE TITLE
[BUG] Webkit is not working with Monaco editor

### ISSUE THREAD

**Issue by kjcho-msft** [microsoft/playwright#9769]
**Context:**
- Playwright Version: 1.15.0
- Operating System: Windows
- Node.js version: 14.15.1
- Browser: WebKit
- 
**Code Snippet**

You can try some samples from https://github.com/microsoft/monaco-editor-samples

or use this:

<!DOCTYPE html>
<html>
	<head>
		<title>browser-amd-editor</title>
		<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
	</head>
	<body>
		<h2>Monaco Editor Sample</h2>
		<div id="container" style="width: 800px; height: 600px; border: 1px solid grey"></div>

		
		<script src="https://unpkg.com/monaco-editor@latest/min/vs/loader.js"></script>
		<script>
			require.config({ paths: { vs: 'https://unpkg.com/monaco-editor@latest/min/vs' } });

			require(['vs/editor/editor.main'], function () {
				var editor = monaco.editor.create(document.getElementById('container'), {
					value: ['function x() {', '\tconsole.log("Hello world!");', '}'].join('\n'),
					language: 'javascript'
				});
			});
		</script>
	</body>
</html>

**Describe the bug**
Webkit would freeze when trying to load/initialize a Monaco editor instance.
And eventually you would see this error below:

WebKitWebProcess.exe

[Main Instruction]
WebKitWebProcess.exe has stopped working

[Content]
A problem caused the program to stop working correctly. Windows will close the program and notify you if a solution is available.

**Comment by kjcho-msft** [comment]
@fujii @yury-s do you have any suggestion on how to address this? Our E2E tests for Webkit browser  are blocked by this issue.

**Comment by yury-s** [comment]
> Do I need to build a custom Webkit browser with that change?

Yes.

> If so, I am not sure how it will work for our pipelines. Can you expose it as a browser setting?

I'll try adding a command line flag that would allow to disable accelerated compositing when launching the browser (will call WKPreferencesSetAcceleratedCompositingEnabled internally).

**Comment by yury-s** [comment]
@kjcho-msft the browser patch has landed and should be available in @next version of playwright, you can pass the new flag to disable accelearated compositing in webkit on windows like below. The example above worked with the flag just fine.


```js
  const browser = await playwright.webkit.launch({
    args: ['--disable-accelerated-compositing']
  });
```

**Comment by kjcho-msft** [comment]
@yury-s It looks like the fix will be in v1.17. Do you have any ETA when it will be released?
